src/sample_cpp/CMakeLists.txt

```
@@ -24,10 +24,12 @@ add_executable(lcevc_dec::sample ALIAS lcevc_dec_sample)
 install(TARGETS "lcevc_dec_sample" DESTINATION "bin")
 
 if(VN_SDK_SAMPLE_SOURCE)
-    install(FILES ${SOURCES} DESTINATION "sample_cpp")
-    install(FILES "standalone/README.md" DESTINATION "sample_cpp")
-    install(FILES "standalone/CMakeLists.txt" DESTINATION "sample_cpp")
-    install(FILES "standalone/conanfile.py" DESTINATION "sample_cpp")
-    install(FILES "standalone/requirements.txt" DESTINATION "sample_cpp")
-    install(DIRECTORY "standalone/cmake" DESTINATION "sample_cpp")
+   include(GNUInstallDirs)
```

```suggestion
    include(GNUInstallDirs)
```

Very recently, we've introduced formatting (including cmake formatting) as a pre-commit git hook, so this would now get caught automatically.